css/templates/e-commerce/page.css

```diff
@@ -5,51 +5,265 @@
 @import 'page-vars';
 
 body {
-  font-size: var(--narrow-body-font-size);
-}
-
-@media (--breakpoint-md) {
-  body {
-    font-size: var(--wide-body-font-size);
-  }
+  font-size: var(--body-font-size);
+  line-height: initial;
 }
 
 /* TYPOGRAPHY */
 .h3 {
   color: var(--color-primary);
+}
+
+.h7 {
+  font-size: var(--h7);
+}
+
+h1, h2, h3, h4, h5, h6, h7,
+.h1, .h2, .h3, .h4, .h5, .h6, .h7 {
   line-height: normal;
-  padding-bottom: var(--space-1);
+  letter-spacing: 0;
+  font-family: var(--title-font);
+  text-transform: uppercase;
+  font-weight: bold;
+  color: var(--color-black);
+}
+
+.mb5 {
+  margin-bottom: var(--space-6);
+}
+
+.pb5 {
+  padding-bottom: var(--space-6);
+}
+
+@media (--breakpoint-md) {
+  .md-h1 {
+    font-size: var(--h1);
+  }
+
+  .md-h2 {
+    font-size: var(--h2);
+  }
+
+  .md-h3 {
+    font-size: var(--h3);
+  }
+
+  .md-h4 {
+    font-size: var(--h4);
+  }
+
+  .md-h5 {
+    font-size: var(--h5);
+  }
+
+  .md-h6 {
+    font-size: var(--h6);
+  }
+
+  .md-h7 {
+    font-size: var(--h7);
+  }
+
+  .md-px4 {
+    padding-left: var(--space-4);
+    padding-right: var(--space-4);
+  }
+
+  .md-pt5 {
+    padding-top: var(--space-5);
+  }
+
+  .md-pl5 {
+    padding-left: var(--space-5);
+  }
+
+  .md-pt6 {
+    padding-top: var(--space-6);
+  }
+
+  .md-pl7 {
+    padding-left: var(--space-7);
+  }
+
+  .md-pr7 {
+    padding-right: var(--space-7);
+  }
+
+  .md-px7 {
+    padding-left: var(--space-7);
+    padding-right: var(--space-7);
+  }
 }
 
 
 /* ELEMENTS */
+hr {
+  width: calc(100% + 2 * var(--space-3));
+  height: 1px;
+  background-color: var(--color-separator);
+  border: none;
+  margin: 0 -var(--space-3);
+}
+
+@media (--breakpoint-md) {
+  hr {
+    width: 100%;
+    margin: 0;
+  }
+}
+
+.commerce-select-wrapper {
+  display: inline-block;
+  position: relative;
+  padding-right: 20px;
+}
+
+.commerce-select-wrapper::after {
+  position: absolute;
+  right: 0;
+  top: 8px;
+  content: '';
+  display: block;
+  width: 0;
+  height: 0;
+  border-left: 5px solid transparent;
+  border-right: 5px solid transparent;
+  border-top: 5px solid var(--color-primary);
+}
+
+.commerce-select {
+  -webkit-appearance: none;
+  border: none;
+  border-radius: 0;
+  background: none;
+}
+
+.ampstart-btn {
+  display: inline-block;
+}
+
 .ampstart-btn-secondary {
   background-color: var(--color-white);
-  color: var(--color-font-accent);
-  border-color: var(--color-font-accent);
+  color: var(--color-primary);
```

Comment:
This is because on the landing page, the buttons are all styled like secondary in narrow but in wider view, they are styled like a primary button. 

Maybe I could add a class like md-ampstart-btn-primary to achieve this affect? What do you think?